### PR TITLE
python3Packages.websockets: 8.1 -> 9.1

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Update to latest upstream release 9.1

Version 9.1 fixes a security issue introduced in version 8.0.

Change log:
- https://websockets.readthedocs.io/en/stable/changelog.html#id2
- https://websockets.readthedocs.io/en/stable/changelog.html#id3
- https://websockets.readthedocs.io/en/stable/changelog.html#id4
- https://websockets.readthedocs.io/en/stable/changelog.html#id5

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 123283` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>python38Packages.pytest-sanic</li>
    <li>python39Packages.pytest-sanic</li>
  </ul>
</details>
<details>
  <summary>31 packages failed to build:</summary>
  <ul>
    <li>dyndnsc</li>
    <li>home-assistant</li>
    <li>python38Packages.aioambient</li>
    <li>python38Packages.amqtt</li>
    <li>python38Packages.entrance</li>
    <li>python38Packages.entrance-with-router-features</li>
    <li>python38Packages.hbmqtt</li>
    <li>python38Packages.json-logging</li>
    <li>python38Packages.pyppeteer</li>
    <li>python38Packages.qiskit</li>
    <li>python38Packages.qiskit-ibmq-provider</li>
    <li>python38Packages.roombapy</li>
    <li>python38Packages.sanic</li>
    <li>python38Packages.sanic-auth</li>
    <li>python38Packages.sanic-testing</li>
    <li>python38Packages.simplisafe-python</li>
    <li>python39Packages.aioambient</li>
    <li>python39Packages.amqtt</li>
    <li>python39Packages.entrance</li>
    <li>python39Packages.entrance-with-router-features</li>
    <li>python39Packages.json-logging</li>
    <li>python39Packages.pyppeteer</li>
    <li>python39Packages.qiskit</li>
    <li>python39Packages.qiskit-ibmq-provider</li>
    <li>python39Packages.roombapy</li>
    <li>python39Packages.sanic</li>
    <li>python39Packages.sanic-auth</li>
    <li>python39Packages.sanic-testing</li>
    <li>python39Packages.simplisafe-python</li>
    <li>theharvester</li>
    <li>urlwatch</li>
  </ul>
</details>
<details>
  <summary>56 packages built:</summary>
  <ul>
    <li>appdaemon</li>
    <li>datasette (python38Packages.datasette)</li>
    <li>grab-site</li>
    <li>onionshare</li>
    <li>onionshare-gui</li>
    <li>python38Packages.PyRMVtransport</li>
    <li>python38Packages.aiopylgtv</li>
    <li>python38Packages.authcaptureproxy</li>
    <li>python38Packages.axis</li>
    <li>python38Packages.denonavr</li>
    <li>python38Packages.dependency-injector</li>
    <li>python38Packages.elmax</li>
    <li>python38Packages.fastapi</li>
    <li>python38Packages.flask-socketio</li>
    <li>python38Packages.homematicip</li>
    <li>python38Packages.httpcore</li>
    <li>python38Packages.httpx</li>
    <li>python38Packages.ipfshttpclient</li>
    <li>python38Packages.pylitterbot</li>
    <li>python38Packages.pytest-httpx</li>
    <li>python38Packages.python-binance</li>
    <li>python38Packages.python-engineio</li>
    <li>python38Packages.python-engineio_3</li>
    <li>python38Packages.python-socketio</li>
    <li>python38Packages.python-socketio_4</li>
    <li>python38Packages.respx</li>
    <li>python38Packages.slack-sdk</li>
    <li>python38Packages.teslajsonpy</li>
    <li>python38Packages.uvicorn</li>
    <li>python38Packages.websockets</li>
    <li>python39Packages.PyRMVtransport</li>
    <li>python39Packages.aiopylgtv</li>
    <li>python39Packages.authcaptureproxy</li>
    <li>python39Packages.axis</li>
    <li>python39Packages.datasette</li>
    <li>python39Packages.denonavr</li>
    <li>python39Packages.dependency-injector</li>
    <li>python39Packages.elmax</li>
    <li>python39Packages.fastapi</li>
    <li>python39Packages.flask-socketio</li>
    <li>python39Packages.homematicip</li>
    <li>python39Packages.httpcore</li>
    <li>python39Packages.httpx</li>
    <li>python39Packages.ipfshttpclient</li>
    <li>python39Packages.pylitterbot</li>
    <li>python39Packages.pytest-httpx</li>
    <li>python39Packages.python-binance</li>
    <li>python39Packages.python-engineio</li>
    <li>python39Packages.python-engineio_3</li>
    <li>python39Packages.python-socketio</li>
    <li>python39Packages.python-socketio_4</li>
    <li>python39Packages.respx</li>
    <li>python39Packages.slack-sdk</li>
    <li>python39Packages.teslajsonpy</li>
    <li>python39Packages.uvicorn</li>
    <li>python39Packages.websockets</li>
  </ul>
</details>

**Comment by fabaff** [comment]
- python3Packages.pyppeteer: https://github.com/NixOS/nixpkgs/pull/124201

**Comment by jonringer** [comment]
@fabaff my notifications are like... 50% your PRs :)

**Comment by jonringer** [comment]
other than the sanic regressions, everything else lgtm

still would like for some time for them to respond.

**Comment by risicle** [comment]
I'm also getting a failure from `simplisafe-python` on macos 10.15:

```
ERROR: Could not find a version that satisfies the requirement websockets<9.0,>=8.1 (from simplisafe-python)
ERROR: No matching distribution found for websockets<9.0,>=8.1
```

**Comment by risicle** [comment]
The constant-time comparison should be fairly trivial to backport for stable branch(es).

https://github.com/aaugustin/websockets/commit/547a26b685d08cac0aa64e5e65f7867ac0ea9bc0

**Comment by risicle** [comment]
I don't know if a new CVE will be assigned for the issue fixed here, but https://nvd.nist.gov/vuln/detail/CVE-2018-1000518 is the CVE for when this happened in the 4->5 days. Guess we call this CVE-2018-1000518-redux.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 123283` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>

  <summary>5 packages failed to build and already failed to build on hydra master:</summary>
  <ul>
<li>python38Packages.qiskit: log was empty</li>
<li>python38Packages.qiskit-ibmq-provider: log was empty</li>
<li>python39Packages.qiskit: log was empty</li>
<li>python39Packages.qiskit-ibmq-provider: log was empty</li>
<li>theharvester: log was empty</li>
  </ul>
</details>
<details>

  <summary>97 packages failed to build and are new build failures:</summary>
  <ul>
<li>appdaemon: log was empty</li>
<li>chia: log was empty</li>
<li>datasette: log was empty</li>
<li>(python38Packages.datasette): log was empty</li>
<li>dyndnsc: log was empty</li>
<li>esphome: log was empty</li>
<li>grab-site: log was empty</li>
<li>home-assistant: log was empty</li>
<li>onionshare: log was empty</li>
<li>onionshare-gui: log was empty</li>
<li>opsdroid: log was empty</li>
<li>platformio: log was empty</li>
<li>python38Packages.PyRMVtransport: log was empty</li>
<li>python38Packages.aioambient: log was empty</li>
<li>python38Packages.aiopylgtv: log was empty</li>
<li>python38Packages.amqtt: log was empty</li>
<li>python38Packages.authcaptureproxy: log was empty</li>
<li>python38Packages.axis: log was empty</li>
<li>python38Packages.denonavr: log was empty</li>
<li>python38Packages.dependency-injector: log was empty</li>
<li>python38Packages.elmax: log was empty</li>
<li>python38Packages.entrance: log was empty</li>
<li>python38Packages.entrance-with-router-features: log was empty</li>
<li>python38Packages.envoy-reader: log was empty</li>
<li>python38Packages.fastapi: log was empty</li>
<li>python38Packages.flask-socketio: log was empty</li>
<li>python38Packages.ha-philipsjs: log was empty</li>
<li>python38Packages.hbmqtt: log was empty</li>
<li>python38Packages.homematicip: log was empty</li>
<li>python38Packages.httpcore: log was empty</li>
<li>python38Packages.httpx: log was empty</li>
<li>python38Packages.ipfshttpclient: log was empty</li>
<li>python38Packages.json-logging: log was empty</li>
<li>python38Packages.mattermostdriver: log was empty</li>
<li>python38Packages.pylitterbot: log was empty</li>
<li>python38Packages.pyppeteer: log was empty</li>
<li>python38Packages.pytest-httpx: log was empty</li>
<li>python38Packages.pytest-sanic: log was empty</li>
<li>python38Packages.python-binance: log was empty</li>
<li>python38Packages.python-engineio: log was empty</li>
<li>python38Packages.python-engineio_3: log was empty</li>
<li>python38Packages.python-socketio: log was empty</li>
<li>python38Packages.python-socketio_4: log was empty</li>
<li>python38Packages.respx: log was empty</li>
<li>python38Packages.roombapy: log was empty</li>
<li>python38Packages.sanic: log was empty</li>
<li>python38Packages.sanic-auth: log was empty</li>
<li>python38Packages.sanic-testing: log was empty</li>
<li>python38Packages.simplisafe-python: log was empty</li>
<li>python38Packages.slack-sdk: log was empty</li>
<li>python38Packages.systembridge: <a href="https://termbin.com/sspr">plain log</a>  |  <a href="https://htmlpreview.github.io/?https://gist.githubusercontent.com/SuperSandroBot/293e16608191c16e4e84621d18343e0f/raw/index.html">pretty log </a></li>
<li>python38Packages.teslajsonpy: log was empty</li>
<li>python38Packages.uvicorn: log was empty</li>
<li>python38Packages.websockets: log was empty</li>
<li>python39Packages.PyRMVtransport: log was empty</li>
<li>python39Packages.aioambient: log was empty</li>
<li>python39Packages.aiopylgtv: log was empty</li>
<li>python39Packages.amqtt: log was empty</li>
<li>python39Packages.authcaptureproxy: log was empty</li>
<li>python39Packages.axis: log was empty</li>
<li>python39Packages.datasette: log was empty</li>
<li>python39Packages.denonavr: log was empty</li>
<li>python39Packages.dependency-injector: log was empty</li>
<li>python39Packages.elmax: log was empty</li>
<li>python39Packages.entrance: log was empty</li>
<li>python39Packages.entrance-with-router-features: log was empty</li>
<li>python39Packages.envoy-reader: log was empty</li>
<li>python39Packages.fastapi: log was empty</li>
<li>python39Packages.flask-socketio: log was empty</li>
<li>python39Packages.ha-philipsjs: log was empty</li>
<li>python39Packages.homematicip: log was empty</li>
<li>python39Packages.httpcore: log was empty</li>
<li>python39Packages.httpx: log was empty</li>
<li>python39Packages.ipfshttpclient: log was empty</li>
<li>python39Packages.json-logging: log was empty</li>
<li>python39Packages.mattermostdriver: log was empty</li>
<li>python39Packages.pylitterbot: log was empty</li>
<li>python39Packages.pyppeteer: log was empty</li>
<li>python39Packages.pytest-httpx: log was empty</li>
<li>python39Packages.pytest-sanic: log was empty</li>
<li>python39Packages.python-binance: log was empty</li>
<li>python39Packages.python-engineio: log was empty</li>
<li>python39Packages.python-engineio_3: log was empty</li>
<li>python39Packages.python-socketio: log was empty</li>
<li>python39Packages.python-socketio_4: log was empty</li>
<li>python39Packages.respx: log was empty</li>
<li>python39Packages.roombapy: log was empty</li>
<li>python39Packages.sanic: log was empty</li>
<li>python39Packages.sanic-auth: log was empty</li>
<li>python39Packages.sanic-testing: log was empty</li>
<li>python39Packages.simplisafe-python: log was empty</li>
<li>python39Packages.slack-sdk: log was empty</li>
<li>python39Packages.systembridge: log was empty</li>
<li>python39Packages.teslajsonpy: log was empty</li>
<li>python39Packages.uvicorn: log was empty</li>
<li>python39Packages.websockets: log was empty</li>
<li>urlwatch: log was empty</li>
  </ul>
</details>

**Comment by fabaff** [comment]
`sanic-testing` (https://github.com/sanic-org/sanic-testing/commit/7814f08076e3618feb739fd18c29500ab92802e4) and `sanic` (https://github.com/sanic-org/sanic/pull/2154) and both in the works.

File: pkgs/applications/blockchains/chia/default.nix [hunk 1]
```diff
@@ -14,6 +18,12 @@ python3Packages.buildPythonApplication rec {
   patches = [
     # tweak version requirements to what's available in Nixpkgs
     ./dependencies.patch
+    # Allow later websockets release, https://github.com/Chia-Network/chia-blockchain/pull/6304
+    (fetchpatch {
+      name = "later-websockets.patch";
+      url = "https://github.com/Chia-Network/chia-blockchain/commit/a188f161bf15a30e8e2efc5eec824e53e2a98a5b.patch";
+      sha256 = "0rws7fhxmca3d5w0bkygx8scvzah3yvb3yfhn05qmp73mn3pmcb1";
+    })
```
Comment (lourkeur) [inline pkgs/applications/blockchains/chia/default.nix:26]: something is wrong with this patch, I get
```
error: hash mismatch in fixed-output derivation '/nix/store/5fg0vkxg3r5awc2gw338k23lfp4hn4r4-later-websockets.patch.drv':
         specified: sha256-YbF6h63j3IoLsND5sbYfUP3NNOrPzwV4aUOx2qE7mmc=
            got:    sha256-Gp0HwOGXHV2rxrbyZTGCu3+WgC3snmjNRyLWSRqUuOg=
error: 1 dependencies of derivation '/nix/store/a8yw9l461g6jia37dc2lsfn3s54qp7p1-chia-1.1.7.drv' failed to build
```

**Review by bbjubjub2494 — CHANGES_REQUESTED** [review]

File: pkgs/applications/blockchains/chia/default.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/applications/blockchains/chia/default.nix]: Fixed, thanks.

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 123283` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>7 packages failed to build:</summary>
  <ul>
    <li>home-assistant</li>
    <li>python38Packages.hbmqtt</li>
    <li>python38Packages.simplisafe-python</li>
    <li>python38Packages.systembridge</li>
    <li>python39Packages.simplisafe-python</li>
    <li>python39Packages.slack-sdk</li>
    <li>python39Packages.systembridge</li>
  </ul>
</details>
<details>
  <summary>98 packages built:</summary>
  <ul>
    <li>appdaemon</li>
    <li>datasette (python38Packages.datasette)</li>
    <li>dyndnsc</li>
    <li>esphome</li>
    <li>grab-site</li>
    <li>lexicon</li>
    <li>onionshare</li>
    <li>onionshare-gui</li>
    <li>opsdroid</li>
    <li>platformio</li>
    <li>python38Packages.PyRMVtransport</li>
    <li>python38Packages.aioambient</li>
    <li>python38Packages.aiopylgtv</li>
    <li>python38Packages.amqtt</li>
    <li>python38Packages.authcaptureproxy</li>
    <li>python38Packages.axis</li>
    <li>python38Packages.denonavr</li>
    <li>python38Packages.dependency-injector</li>
    <li>python38Packages.elmax</li>
    <li>python38Packages.entrance</li>
    <li>python38Packages.entrance-with-router-features</li>
    <li>python38Packages.envoy-reader</li>
    <li>python38Packages.fastapi</li>
    <li>python38Packages.flask-socketio</li>
    <li>python38Packages.ha-philipsjs</li>
    <li>python38Packages.homematicip</li>
    <li>python38Packages.httpcore</li>
    <li>python38Packages.httpx</li>
    <li>python38Packages.ipfshttpclient</li>
    <li>python38Packages.json-logging</li>
    <li>python38Packages.mattermostdriver</li>
    <li>python38Packages.pylitterbot</li>
    <li>python38Packages.pyppeteer</li>
    <li>python38Packages.pytest-httpx</li>
    <li>python38Packages.pytest-sanic</li>
    <li>python38Packages.python-binance</li>
    <li>python38Packages.python-engineio</li>
    <li>python38Packages.python-engineio_3</li>
    <li>python38Packages.python-socketio</li>
    <li>python38Packages.python-socketio_4</li>
    <li>python38Packages.qiskit</li>
    <li>python38Packages.qiskit-ibmq-provider</li>
    <li>python38Packages.respx</li>
    <li>python38Packages.roombapy</li>
    <li>python38Packages.sanic</li>
    <li>python38Packages.sanic-auth</li>
    <li>python38Packages.sanic-testing</li>
    <li>python38Packages.slack-sdk</li>
    <li>python38Packages.sunpy</li>
    <li>python38Packages.teslajsonpy</li>
    <li>python38Packages.uvicorn</li>
    <li>python38Packages.websockets</li>
    <li>python38Packages.zeep</li>
    <li>python39Packages.PyRMVtransport</li>
    <li>python39Packages.aioambient</li>
    <li>python39Packages.aiopylgtv</li>
    <li>python39Packages.amqtt</li>
    <li>python39Packages.authcaptureproxy</li>
    <li>python39Packages.axis</li>
    <li>python39Packages.datasette</li>
    <li>python39Packages.denonavr</li>
    <li>python39Packages.dependency-injector</li>
    <li>python39Packages.elmax</li>
    <li>python39Packages.entrance</li>
    <li>python39Packages.entrance-with-router-features</li>
    <li>python39Packages.envoy-reader</li>
    <li>python39Packages.fastapi</li>
    <li>python39Packages.flask-socketio</li>
    <li>python39Packages.ha-philipsjs</li>
    <li>python39Packages.homematicip</li>
    <li>python39Packages.httpcore</li>
    <li>python39Packages.httpx</li>
    <li>python39Packages.ipfshttpclient</li>
    <li>python39Packages.json-logging</li>
    <li>python39Packages.mattermostdriver</li>
    <li>python39Packages.pylitterbot</li>
    <li>python39Packages.pyppeteer</li>
    <li>python39Packages.pytest-httpx</li>
    <li>python39Packages.pytest-sanic</li>
    <li>python39Packages.python-binance</li>
    <li>python39Packages.python-engineio</li>
    <li>python39Packages.python-engineio_3</li>
    <li>python39Packages.python-socketio</li>
    <li>python39Packages.python-socketio_4</li>
    <li>python39Packages.qiskit</li>
    <li>python39Packages.qiskit-ibmq-provider</li>
    <li>python39Packages.respx</li>
    <li>python39Packages.roombapy</li>
    <li>python39Packages.sanic</li>
    <li>python39Packages.sanic-auth</li>
    <li>python39Packages.sanic-testing</li>
    <li>python39Packages.sunpy</li>
    <li>python39Packages.teslajsonpy</li>
    <li>python39Packages.uvicorn</li>
    <li>python39Packages.websockets</li>
    <li>python39Packages.zeep</li>
    <li>theharvester</li>
    <li>urlwatch</li>
  </ul>
</details>

**Review by bbjubjub2494 — COMMENTED** [review]
As far as Onionshare is concerned, :heavy_check_mark:.

As far as Chia is concerned, the pytestCheckPhase now takes hours and fails. Looking into it. Did anyone get it to run?

**Comment by fabaff** [comment]
> As far as Chia is concerned, the pytestCheckPhase now takes hours and fails. Looking into it. Did anyone get it to run?

I'm testing without chia as those tests take 2.5 h to run for me.

**Comment by lourkeur** [comment]
> > As far as Chia is concerned, the pytestCheckPhase now takes hours and fails. Looking into it. Did anyone get it to run?
> 
> I'm testing without chia as those tests take 2.5 h to run for me.

I figured that if you remove pytest-asyncio it goes back to <30 minutes. short term we can go there. Long term I'm going to investigate further.

**Review by bbjubjub2494 — CHANGES_REQUESTED** [review]
commit message: chai -> chia

**Comment by fabaff** [comment]
> commit message: chai -> chia

Thanks. I really hope that I'm not ordering a "Masala chia" next time.

**Review by bbjubjub2494 — APPROVED** [review]
LGTM as far as packages I have a hand in are concerned.

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 123283` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>6 packages failed to build:</summary>
  <ul>
    <li>home-assistant</li>
    <li>python38Packages.hbmqtt</li>
    <li>python38Packages.simplisafe-python</li>
    <li>python38Packages.systembridge</li>
    <li>python39Packages.simplisafe-python</li>
    <li>python39Packages.systembridge</li>
  </ul>
</details>
<details>
  <summary>100 packages built:</summary>
  <ul>
    <li>appdaemon</li>
    <li>chia</li>
    <li>datasette (python38Packages.datasette)</li>
    <li>dyndnsc</li>
    <li>esphome</li>
    <li>grab-site</li>
    <li>lexicon</li>
    <li>onionshare</li>
    <li>onionshare-gui</li>
    <li>opsdroid</li>
    <li>platformio</li>
    <li>python38Packages.PyRMVtransport</li>
    <li>python38Packages.aioambient</li>
    <li>python38Packages.aiopylgtv</li>
    <li>python38Packages.amqtt</li>
    <li>python38Packages.authcaptureproxy</li>
    <li>python38Packages.axis</li>
    <li>python38Packages.denonavr</li>
    <li>python38Packages.dependency-injector</li>
    <li>python38Packages.elmax</li>
    <li>python38Packages.entrance</li>
    <li>python38Packages.entrance-with-router-features</li>
    <li>python38Packages.envoy-reader</li>
    <li>python38Packages.fastapi</li>
    <li>python38Packages.flask-socketio</li>
    <li>python38Packages.ha-philipsjs</li>
    <li>python38Packages.homematicip</li>
    <li>python38Packages.httpcore</li>
    <li>python38Packages.httpx</li>
    <li>python38Packages.ipfshttpclient</li>
    <li>python38Packages.json-logging</li>
    <li>python38Packages.mattermostdriver</li>
    <li>python38Packages.pylitterbot</li>
    <li>python38Packages.pyppeteer</li>
    <li>python38Packages.pytest-httpx</li>
    <li>python38Packages.pytest-sanic</li>
    <li>python38Packages.python-binance</li>
    <li>python38Packages.python-engineio</li>
    <li>python38Packages.python-engineio_3</li>
    <li>python38Packages.python-socketio</li>
    <li>python38Packages.python-socketio_4</li>
    <li>python38Packages.qiskit</li>
    <li>python38Packages.qiskit-ibmq-provider</li>
    <li>python38Packages.respx</li>
    <li>python38Packages.roombapy</li>
    <li>python38Packages.sanic</li>
    <li>python38Packages.sanic-auth</li>
    <li>python38Packages.sanic-testing</li>
    <li>python38Packages.slack-sdk</li>
    <li>python38Packages.sunpy</li>
    <li>python38Packages.teslajsonpy</li>
    <li>python38Packages.uvicorn</li>
    <li>python38Packages.websockets</li>
    <li>python38Packages.zeep</li>
    <li>python39Packages.PyRMVtransport</li>
    <li>python39Packages.aioambient</li>
    <li>python39Packages.aiopylgtv</li>
    <li>python39Packages.amqtt</li>
    <li>python39Packages.authcaptureproxy</li>
    <li>python39Packages.axis</li>
    <li>python39Packages.datasette</li>
    <li>python39Packages.denonavr</li>
    <li>python39Packages.dependency-injector</li>
    <li>python39Packages.elmax</li>
    <li>python39Packages.entrance</li>
    <li>python39Packages.entrance-with-router-features</li>
    <li>python39Packages.envoy-reader</li>
    <li>python39Packages.fastapi</li>
    <li>python39Packages.flask-socketio</li>
    <li>python39Packages.ha-philipsjs</li>
    <li>python39Packages.homematicip</li>
    <li>python39Packages.httpcore</li>
    <li>python39Packages.httpx</li>
    <li>python39Packages.ipfshttpclient</li>
    <li>python39Packages.json-logging</li>
    <li>python39Packages.mattermostdriver</li>
    <li>python39Packages.pylitterbot</li>
    <li>python39Packages.pyppeteer</li>
    <li>python39Packages.pytest-httpx</li>
    <li>python39Packages.pytest-sanic</li>
    <li>python39Packages.python-binance</li>
    <li>python39Packages.python-engineio</li>
    <li>python39Packages.python-engineio_3</li>
    <li>python39Packages.python-socketio</li>
    <li>python39Packages.python-socketio_4</li>
    <li>python39Packages.qiskit</li>
    <li>python39Packages.qiskit-ibmq-provider</li>
    <li>python39Packages.respx</li>
    <li>python39Packages.roombapy</li>
    <li>python39Packages.sanic</li>
    <li>python39Packages.sanic-auth</li>
    <li>python39Packages.sanic-testing</li>
    <li>python39Packages.slack-sdk</li>
    <li>python39Packages.sunpy</li>
    <li>python39Packages.teslajsonpy</li>
    <li>python39Packages.uvicorn</li>
    <li>python39Packages.websockets</li>
    <li>python39Packages.zeep</li>
    <li>theharvester</li>
    <li>urlwatch</li>
  </ul>
</details>

**Comment by fabaff** [comment]
- `python3Packages.systembridge`: no clear why there is a failure (ERROR: Could not find a version that satisfies the requirement websockets>=9.0.2 (from systembridge))
- `python3Packages.simplisafe-python`: https://github.com/NixOS/nixpkgs/pull/125445/commits/a373431cd7d0d933cf357708c1ae44e0857e8110
- `python38Packages.hbmqtt`: unmaintained (-> `amqtt` is a maintained drop-in replacement) 

I will merge this and deal with the remaining issues later.

**Pull request merged**